matlab/lib/+Ice/AlreadyRegisteredException.m

```diff
@@ -20,23 +19,20 @@
         id char
     end
     methods
-        function obj = AlreadyRegisteredException(ice_exid, ice_exmsg, kindOfObject, id)
-            if nargin <= 2
+        % Convenience constructor without an errID or what message.
+        function obj = AlreadyRegisteredException(kindOfObject, id)
+            if nargin == 0 % default constructor
```

Comment:
A general pattern/guideline in MATLAB is to provide a default constructor for all classes.  I don't think it's ever used for our local exceptions.